cmd/relayproxy/service/gofeatureflag.go

```diff
@@ -142,6 +142,19 @@ func initRetriever(c *config.RetrieverConf) (retriever.Retriever, error) {
 			RepositorySlug: c.RepositorySlug,
 			Timeout:        retrieverTimeout,
 		}, nil
+	case config.BitbucketRetriever:
+		return &bitbucketretriever.Retriever{
+			RepositorySlug: c.RepositorySlug,
+			Branch: func() string {
+				if c.Branch == "" {
+					return config.DefaultRetriever.GitBranch
+				}
+				return c.Branch
+			}(),
```

Comment:
**suggestion:** we can simplify that part because the default value is already handled in the `retriever` [here](https://github.com/thomaspoignant/go-feature-flag/blob/9ef1547c91b479b4464c539209edd06f50677af1/retriever/bitbucketretriever/retriever.go#L36).

```suggestion
			Branch: c.Branch,
```